.husky/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# git hook to run a command before actually committing when running `git commit`:
+#
+
+ORANGE='\033[0;33m'
+NC='\033[0m' # No Color
+
+log() {
+  printf "${ORANGE}[pre-commit]${NC} $1\n"
+}
+
+log "Running custom pre-commit hook defined in ./husky/pre-commit"
+
+# Prevent committing to "main"-branch:
+if [ "$(git rev-parse --abbrev-ref HEAD)" = "main" ]; then
```

Comment:
@jannisvisser @arsforza This will not work because IBF does not have a [`main`](https://github.com/rodekruis/IBF-system/tree/main) branch. It does have a [`master`](https://github.com/rodekruis/IBF-system/tree/master) branch which we could use. Or create a `main` branch and use that instead of `master`.

Reply:
good point I'll correct it